pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -1745,3 +1577,15 @@ def test_processing_order(self):
         result = pd.to_datetime(300 * 365, unit='D', origin='1870-01-01')
         expected = Timestamp('2169-10-20 00:00:00')
         assert result == expected
+
+
+def test_to_datetime1():
+    actual = pd.to_datetime(datetime(2008, 1, 15))
```

Comment:
can you add a refernce for this test (e.g. what is it testing)

can you put it near other tests that are similar